.github/workflows/dab-consumers.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           FROM ghcr.io/ansible/awx_devel:devel
           COPY . /opt/dab/
           RUN pip install -e /opt/dab/
+          RUN pip install jq
```

Comment:
Is this still necessary or was it just part of your testing?

Reply:
Not related to my change, just trying to get the awx job to pass..